scripts/release/xenon-survival.R

```diff
@@ -0,0 +1,109 @@
+create_survival_object <- function(){
+
+  dsSurvivalClient::ds.Surv(
+    time='survival$time',
+    event = 'survival$status',
+    objectname='surv_object')
+
+}
+
+verify_survival_class <- function(){
+
+  surv_class <- ds.class("surv_object")
+
+  if(surv_class$armadillo == "Surv"){
+    cli_alert_success("ds.survival passed")
+  } else{
+    cli_alert_danger("ds.survival failed")
+    exit_test("ds.survival did not create a serverside object with the expected class")
+  }
+
+}
+
+verify_cox_output <- function(){
+
+  cox_output <- dsSurvivalClient::ds.coxph.SLMA(formula = 'surv_object~survival$age')
+  expected_names <- c("call", "fail", "na.action", "n", "loglik", "nevent", "coefficients",
+                      "conf.int", "logtest", "sctest", "rsq", "waldtest", "used.robust", "concordance")
+
+  if(identical(names(cox_output$armadillo), expected_names)){
+    cli_alert_success("ds.coxph.SLMA passed")
+  } else{
```

Comment:
space between `else` and `{` (same below, and above)

Reply:
Done